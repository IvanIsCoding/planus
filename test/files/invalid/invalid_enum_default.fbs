enum Foo: uint8 { X = 17 }

table Bar {
  x: Foo = 18;
}