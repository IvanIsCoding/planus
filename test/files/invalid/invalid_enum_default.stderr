[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_enum_default.fbs:4:12
  [0m[34mâ”‚[0m
[0m[34m4[0m [0m[34mâ”‚[0m   x: Foo = [0m[31m18[0m;
  [0m[34mâ”‚[0m            [0m[31m^^[0m [0m[31mExpression does not map to a valid enum variant[0m

Error: Bailing because of errors
